source/MRViewer/MRViewerIO.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include "exports.h"
+#include "MRMesh/MRProgressCallback.h"
+#include <filesystem>
+#include <cstring>
+
+namespace MR
+{
+class Object;
+
+/**
+ * \brief save visual object (mesh, lines, points or voxels) to file
+ * \param callback - callback function to set progress (for progress bar)
+ * \return empty string if no error or error text
+ */
+MRVIEWER_API std::string saveObjectToFile( const Object& obj, const std::filesystem::path& filename,
```

Comment:
Let us better return `tl::expected<void, std::string>` as in MRMeshSave

Reply:
fixed